### PR TITLE
Handle errors

### PR DESCRIPTION
**Pull request by nwatson**

### CODE REVIEW

**Comment by nwatson** [comment]
@eam @xaviershay

File: lib/rangeclient.rb [hunk 1]
```diff
@@ -37,6 +40,7 @@ def expand(arg)
     http.use_ssl = @ssl
     req = Net::HTTP::Get.new('/range/list?' + escaped_arg)
     resp = http.request(req)
+    raise QueryException.new(resp.body) unless resp.code == "200"
```
Comment (xaviershay) [inline lib/rangeclient.rb]: nit: more canonical way to do this is `Net::HTTPSuccess === resp` (or `resp.is_a?(Net::HTTPSuccess)` .... they're effectively equivalent), which avoids the ugly 200-is-a-string issue.

**Comment by xaviershay** [comment]
LGTM!

**Comment by nwatson** [comment]
Fixed. Can I get a merge? Not an owner.

**Pull request merged**